templates/blog/post_detail.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ post.title }}</h1>
+  <hr>
+  <p>{{ post.content }}</p>
+  <hr>
+  {{ post.owner.first_name }} {{ post.owner.last_name }} {{ post.created_time }}
+  <hr>
+  {% if post.commentarys %}
```

Comment:
The condition `{% if post.commentarys %}` will always be true because it's checking if the related manager exists, not if there are any comments. Instead, use `{% if post.commentarys.exists %}` to check if there are any comments associated with the post.